docs/src/architecture/hardening.md

```diff
@@ -68,3 +68,13 @@ The host kernel runs on bare metal. The kernel is provided either with Linux ups
 The host kernel hardening is based on Linux `make tinyconfig`. The default `tinyconfig` fails to assertions on NixOS without modifications. Assertions are fixed in `ghaf_host_hardened_baseline` Linux configuration under Ghaf `modules/host/`.
 
 In addition, NixOS (Ghaf baseline dependency) requires several kernel modules that are added to the config or ignored with `allowMissing = true`. As of now, the kernel builds and early boots on Lenovo X1.
+
+### Virtualization Support
+
+The host Virtualization support will add required kernel config dependency to Ghaf baseline by which NixOS have virtualization enabled. It can be enabled using following flag `host.kernel_virtualization_hardening.enable` for Lenovo X1.
```

Comment:
...**the** required kernel config...
...to **the** Ghaf baseline...
...NixOS **has**...
It can be enabled **with the** following...

Reply:
Updated as per review comment.